deployment/apps/servo_service/config.json

```diff
@@ -3,19 +3,19 @@
         {
         "actuator_id":60,
         "channel": 0,
-        "on_pos" : 500,
+        "on_pos" : 3000,
         "off_pos" : 3,
         "mosfet_id" : 1,
-        "on_losening_pos": 200,
-        "off_losening_pos":4000,
-        "servo_delay":1000,
-        "mosfet_delay":1000
+        "on_losening_pos": 4000,
+        "off_losening_pos":200,
+        "servo_delay":100,
+        "mosfet_delay":100
         },
         {
         "actuator_id":61,
         "channel": 1,
-        "on_pos" : 1000,
-        "off_pos" : 10
+        "on_pos" : 3000,
+        "off_pos" : 1000
```

Comment:
Czemu taka diametralna zmiana ?

Reply:
To wsm mogło nie wchodzić do pr, bawiłem się serwem tym tanim na 5v i na pwm=0 kręcił się w kółko